### DISCUSSION TITLE
A link to a discussion that may be worth reading.

### DISCUSSION THREAD

**Discussion by REMC19** [Aries85/LightZone#278] [General]
Hi Masahiro
It may be worth you reading this discussion on Pixls, please read it all the way through - as there is the odd negative comment on there, but you will also see that people do still like the software, but are worried that it was going away.
https://discuss.pixls.us/t/new-lightzone-release-4-2/17120/47
I don't know if you have any interest in working with any other developers, but several appear to frequent Pixls.us and most of them seem to be quite happy with sharing ideas and code between each others projects - so that might be a way to take a bit of the weight of developing the program all by yourself away.

**Comment by ktgw0316** [comment]
@REMC19  Hi, I've just post a [comment](https://discuss.pixls.us/t/new-lightzone-release-4-2/17120/49?u=ktgw0316) on the PIXLS.US.
